build/packages.targets

```diff
@@ -81,21 +79,21 @@
     <!-- Test and utility packages -->
     <ItemGroup>
         <PackageReference Update="EnvDTE" Version="8.0.1" />
-        <PackageReference Update="FluentAssertions" Version="$(FluentAssertionsVersion)" />
+        <PackageReference Update="FluentAssertions" Version="4.19.4" />
```

Comment:
May I know why not changing the value of property `FluentAssertionsVersion`?And also the `TestSDKVersion `and `MoqVersion`
I checked on the NuGet.org, FluentAssertions and Microsoft.NET.Test.Sdk have a newer stable version than the specified version, so I'm wondering how did you figure out the version? Thanks.

Reply:
> May I know why not changing the value of property FluentAssertionsVersion

It was only used in 1 place. I didn't see the point on having a variable for it. It's declared only in this file.
Similar thing applies to to the other 2. 

> I checked on the NuGet.org, FluentAssertions and Microsoft.NET.Test.Sdk have a newer stable version than the specified version, so I'm wondering how did you figure out the version? Thanks.

FluentAssertions had quite a few breaking changes, so I decided not to update it for now. 
The test SDK had some failures as well. 

I basically focused on updating the product versions more than updating the test package versions.